app/screens/channel_info/channel_info_header.js

```diff
@@ -78,6 +82,30 @@ export default class ChannelInfoHeader extends React.PureComponent {
                         {displayName}
                     </Text>
                 </View>
+                {hasGuests && type === General.GM_CHANNEL &&
```

Comment:
maybe simplify this by creating a function that returns the proper text where you can do

```
<View style={style.section}>
   <FormattedText
     id={id}
     defaultMessage={defaultMessage}
      style={style.header}
    />
</View>
```